potpie/observability/observability/sinks/logfire_sink.py

```
@@ -83,6 +82,5 @@ def shutdown(self, config: ObservabilityConfig) -> None:
                 try:
                     fn()
                     return
-                except Exception as exc:
-                    _logger.debug("logfire %s failed during shutdown: %s", name, exc)
+                except Exception:
                     continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)